python/ml4ir/applications/ranking/tests/test_feature_layer.py

```diff
@@ -135,6 +135,91 @@ def test_categorical_embedding_to_encoding_bilstm_file_truncation(self):
         assert tf.reduce_all(tf.equal(sequence_encoding[3], sequence_encoding[4]))
         assert not tf.reduce_all(tf.equal(sequence_encoding[1], sequence_encoding[4]))
 
+    def test_categorical_embedding_to_encoding_bilstm_sequence_of_words(self):
+        """
+        Check that categorical_embedding_to_encoding_bilstm function does not fail with sequences of word (in a single
+        row).
+        """
+        embedding_size = 32
+        encoding_size = 64
+        feature_info = {
+            "name": "categorical_variable",
+            "feature_layer_info": {
+                "type": "numeric",
+                "fn": "categorical_embedding_to_encoding_bilstm",
+                "args": {
+                    "vocabulary_file": "ml4ir/applications/classification/tests/data/configs/vocabulary/entity_id.csv",
+                    "encoding_type": "bilstm",
+                    "encoding_size": encoding_size,
+                    "embedding_size": embedding_size,
+                    "dropout_rate": 0.2
+                },
+            },
+            "tfrecord_type": SequenceExampleTypeKey.CONTEXT,
+        }
+
+        string_tensor = tf.constant([[['AAA', 'BBB', 'out_of_vocabulary']]])
+
+        categorical_fns.categorical_embedding_to_encoding_bilstm(
+            string_tensor, feature_info, self.file_io
+        )
+
+    def test_categorical_embedding_to_encoding_bilstm_oov_mapping_with_dropout(self):
```

Comment:
@mbrette Wasn't this test already there? Why do I see it in the diff?

Reply:
I don't think so. I added 2 tests. One that use a sequence of words as input (all the other tests were using batches of single word).
The second one, comprehensively go through all the vocabulary terms, which was not there either.

Reply:
Ah. Okay. I see. I misread. My bad. Good check to have.